### PR TITLE
Handle connection errors

### PR DESCRIPTION
**Pull request by hkumarmk**

### CODE REVIEW

File: jiocloud/utils.py [hunk 1]
```diff
@@ -28,22 +29,39 @@ def is_rfc1918(ip_string):
...
+    # will allways return the IP if the server exists
+    ##
+    while True:
+        try:
+            servers = nova_client.servers.list()
+        except Exception as e:
+            print >> sys.stderr, 'Failed on nova list', e
+        else:
+            time.sleep(5)
+            break
```
Comment (sorenh) [inline jiocloud/utils.py]: What is this else-statement supposed to do?

Comment (hkumarmk) [inline jiocloud/utils.py]: if servers list work, just break while loop

Comment (sorenh) [inline jiocloud/utils.py]: try:
         foo()
    except SomeException, e:
        bar()
    except:
        baz()
    else:
        wibble()

If `foo()` raises a SomeException, bar() will be called. If it raises any other exception, baz() will be called. If it does not raise an exception, wibble() is called.

So in your code, if servers.list() succeeds, you sleep for 5 seconds before breaking out of the loop.

Comment (hkumarmk) [inline jiocloud/utils.py]: Ah that was a mistake... will fix it

Comment (hkumarmk) [inline jiocloud/utils.py]: added sleep in the except: section

Comment (hkumarmk) [inline jiocloud/utils.py]: so it became like this.

    while True:
        try:
            servers = nova_client.servers.list()
        except Exception as e:
            print >> sys.stderr, 'Failed on nova list', e
            time.sleep(5)
        else:
            break

**Pull request merged**